microsoft-edge/progressive-web-apps/samples/temperature-converter.md

```diff
@@ -0,0 +1,804 @@
+
+1. At the bottom, click the **Uninstall** button.
+
+   The **Uninstall app from Microsoft Edge on all synced devices?** dialog opens.
+
+1. Select the **Delete app history and data** checkbox, and then click the **Uninstall** button.
+
+
+<!-- ------------------------------ -->
+#### Uninstalling a local app from the Windows Start menu
+
+Note: As an example of OS integration, you could uninstall the app via the Windows Start menu, as follows:
+
+1. Press the **Windows** key.
+
+1. Start typing "temperature converter".
+
+1. Click **Uninstall**:
+
+   ![Windows Start menu showing the Uninstall command for the app](./temperature-converter-images/uninstall-via-start-menu.png)
+
+
+<!-- ------------------------------ -->
+#### Uninstalling a local app by starting from Microsoft Edge
+
+Note: As an alternative, you could uninstall the sample Progressive Web App (PWA) by starting from Microsoft Edge, as follows:
+
+1. In Microsoft Edge, select **Settings and more** (**...**) > **More tools** > **Apps** > **View apps**.
+
+   The **Apps** dialog opens within Microsoft Edge, listing any installed PWAs:
+
+   ![Edge Settings > View apps listing the temperature converter](./temperature-converter-images/settings-view-apps.png)
+
+1. Right-click the **Temperature converter** card, and then click **Manage app**.
+
+   The **All apps / Temperature converter** page opens, at `edge://apps`.
+
+1. At the bottom, click the **Uninstall** button.
+
+Continue with the steps below.
+
+
+<!-- ====================================================================== -->
+## Step 12: Modify the sample
+
+You can modify the sample as a starting point for your own Progressive Web App (PWA).  You'll change the name of the app from "Temperature converter" to "TC", in a working branch of the repo, and then install the modified app locally.
+
+Create a working branch and switch to it, as follows:
+
+1. Open Visual Studio.
+
+   **Open the folder in VS Code**
+
+1. Click the **File** menu, and then click **Open Folder**.
+
+   The **Open Folder** dialog opens.
+
+1. Navigate to the cloned Demos repo folder, such as `C:\Users\localAccount\GitHub\Demos\`, select the **Demos** folder, and then click the **Select Folder** button.
+
+1. In the **Explorer** pane, open `\Demos\temperature-converter\index.html`.
+
+   ![The main branch indicated in lower left of VS Code](./temperature-converter-images/main-branch-in-vs-code.png)
+
+   In the lower left corner of Visual Studio Code, the **main** branch is indicated.
+
+   **Create a working branch and switch to it**
+
+1. In Visual Studio Code, in the **View** menu, select **Terminal**.
+
+1. Press **Ctrl+C**.
+
+   The localhost web server (http-server) stops.  The prompt shows the current path, such as: `PS C:\Users\localAccount\GitHub\Demos\temperature-converter>`
+
+1. If you aren't in that directory already, change to the `/Demos/temperature-converter/` directory, by entering commands such as the following:
```

Comment:
If they were just running the npx server, they will, for sure, already be in the right directory.